### PR TITLE
expression: delete blank line of the builtin_control_vec_generated.go

### PR DESCRIPTION
**Pull request by mmyj**
PCP #12106

### What problem does this PR solve? 

just delete the blank lines of the builtin_control_vec_generated.go
same like #13079

### CODE REVIEW

File: expression/generator/control_vec.go [hunk 1]
```diff
@@ -215,8 +215,7 @@ func (b *builtinIfNull{{ .TypeName }}Sig) vecEval{{ .TypeName }}(input *chunk.Ch
 			result.AppendNull()
 		}
 	}
-	{{ end }}
-	return nil
+	{{ end }}return nil
```
Comment (XuHuaiyu) [inline expression/generator/control_vec.go]: Is this modified by mistake?

Comment (mmyj) [reply expression/generator/control_vec.go]: I will fix it.

**Pull request merged**